### ISSUE TITLE
referencing unreleased schemas

### ISSUE THREAD

**Issue by bendichter** [NeurodataWithoutBorders/pynwb#1125]
The nwb-schema 2.1.0 was released Sept 16. On Sep 26, there were some major changes to the schema, including refactoring of neurodata_types into hdmf-common. The latest release of pynwb f62f52e300a098b161ccca7c1c5fd598feef28e5 references this Sep 26 commit of the schema, even though the changes to the schema were unreleased and the version number is unchanged. This is a problem because we now have nwb files that claim to correspond to the same schema, but in reality are created against different states of nwb-schema.

Immediate solution: Release the Sept 26 commit under `2.2.0`.

Solution going forward: Let's make sure we only link to released commits of nwb-schema, and continue to ensure that all releases are accompanied by version updates. It also might make sense to have intermediate versions be numbered indicating that, such as `2.1.0dev`.

**Comment by bendichter** [comment]
fyi @ln-vidrio

**Comment by oruebel** [comment]
@bendichter I agree that we need to make sure that when changes to the schema happen that we also update the version every time (independent of whether those changes have been officially released). @rly @ajtritt can you help fix the versions?

**Comment by bendichter** [comment]
I think this is fixed, right? @rly, should we close?
